packages/components/src/theme/theme-spectrum/theme-spectrum-overrides.css

```diff
@@ -111,3 +111,8 @@ label[class*='spectrum-Radio'] {
     --dh-color-negative-key-focus-bg
   );
 }
+
+span[class*='spectrum-ToggleSwitch-switch'] {
+  /* increase contrast of switch in off position */
+  background: var(--dh-color-gray-500);
```

Comment:
Should we introduce a `--dh-toggle-switch-bg: var(--dh-color-gray-500)` to make this overridable?

Reply:
yes, and actually should double check it in light mode, as I didn't.

Should be added to theme-dark-components.css and theme-light-components.css with appropriate values, that use that variable here.